### PR TITLE
Improve various translations for de-DE

### PR DESCRIPTION
**Pull request by chausner**
- [x] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs).
- [ ] This pull request is related to an issue.

-----

This fixes and improves a number of translations for the German language. Some contained grammatical errors or typos, others used very uncommon phrasing, yet others I would consider acceptable but I suggest alternative phrasings which fit better in my opinion.

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/3044)

### CODE REVIEW

File: Localization/Resources/de-DE/winget.resw [hunk 1]
```diff
@@ -202,7 +202,7 @@
     <comment>{Locked="{0}"} Error message displayed when the user provides an extra positional argument when none was expected. {0} is a placeholder replaced by the user's extra argument input.</comment>
   </data>
   <data name="FeatureDisabledMessage" xml:space="preserve">
-    <value>Diese Funktion ist zurzeit in Arbeit und kann hat in Zukunft geändert oder entfernt werden. Um es zu aktivieren, bearbeiten Sie Ihre Einstellungen („winget settings“), um die Experimentelle Funktion einzuschließen: „{0}“</value>
+    <value>Diese Funktion ist noch in Entwicklung und kann in Zukunft geändert oder entfernt werden. Um Sie zu aktivieren, bearbeiten Sie Ihre Einstellungen („winget settings“) und fügen Sie die experimentelle Funktion hinzu: „{0}“</value>
```
Comment (SteffoSpieler) [inline Localization/Resources/de-DE/winget.resw:205]: `[...] Um Sie zu aktivieren [...]`
The context is the feature, not the user. So shouldn't it be lowercase `sie`?

Comment (chausner) [inline Localization/Resources/de-DE/winget.resw:205]:
```suggestion
    <value>Diese Funktion ist noch in Entwicklung und kann in Zukunft geändert oder entfernt werden. Um sie zu aktivieren, bearbeiten Sie Ihre Einstellungen („winget settings“) und fügen Sie die experimentelle Funktion hinzu: „{0}“</value>
```

Comment (chausner) [reply Localization/Resources/de-DE/winget.resw]: Good catch, thanks. Fixed it.

**Comment by ryfu-msft** [comment]
Suggestions have been submitted for internal review. Will close this PR once your changes have been checked in.

**Pull request closed without merging**